### DISCUSSION TITLE
neno 记录的 memos 存储的位置

### DISCUSSION THREAD

**Discussion by yiranzhimo** [openneno/neno#21] [Q&A]
作者你好，谢谢你提供了这样的一个开源应用，让每个人可以制作自己的 flomo。

但是，我部署完成以后有一点疑问：neno 把记录的 memo 存储在哪？

**Comment by yiranzhimo** [comment]
> 作者你好，谢谢你提供了这样的一个开源应用，让每个人可以制作自己的 flomo。
> 
> 但是，我部署完成以后有一点疑问：neno 把记录的 memo 存储在哪？

我明确了，在我设置的 github repo 里可以看到 neno 存储的 memos

**Comment by Mran** [comment]
> > 作者你好，谢谢你提供了这样的一个开源应用，让每个人可以制作自己的 flomo。
> > 但是，我部署完成以后有一点疑问：neno 把记录的 memo 存储在哪？
> 
> 我明确了，在我设置的 github repo 里可以看到 neno 存储的 memos

是的，neno利用github进行存储，同样的你可以使用国内的gitee

**Comment by yiranzhimo** [comment]
> > > 作者你好，谢谢你提供了这样的一个开源应用，让每个人可以制作自己的 flomo。
> > > 但是，我部署完成以后有一点疑问：neno 把记录的 memo 存储在哪？
> > 
> > 
> > 我明确了，在我设置的 github repo 里可以看到 neno 存储的 memos
> 
> 是的，neno利用github进行存储，同样的你可以使用国内的gitee

作者好，请问每日回顾和随机漫步后续有上线计划吗？

**Comment by Mran** [comment]
@yiranzhimo 
暂时没做的原因有几个
1. 如果neno有自己的后台服务器，那么可以进行集中处理，但是这和neno不存储用户的数据的初衷背离，需要找到一个合适的方案。
2. 我需要学习nlp、补充一些相关的机器学习的知识。
未来我会把这个功能做上线。
感谢支持。

**Comment by yiranzhimo** [comment]
> @yiranzhimo 暂时没做的原因有几个
> 
> 1. 如果neno有自己的后台服务器，那么可以进行集中处理，但是这和neno不存储用户的数据的初衷背离，需要找到一个合适的方案。
> 2. 我需要学习nlp、补充一些相关的机器学习的知识。
>    未来我会把这个功能做上线。
>    感谢支持。

我从 flomo 找到 neno ，其中的一点原因就是 neno 不存储用户数据，「all rights is on the hand of user」，感谢🙏，，加油学习！
